### ISSUE TITLE
Request updating WDK metadata

### ISSUE THREAD

**Issue by Unpublished** [microsoft/wdkmetadata#29]
I'd like https://github.com/microsoft/wdkmetadata/releases/tag/v0.2.22-experimental to be included in the next `sys` release.

**Comment by mikebattista** [comment]
I just published an update.

**Comment by kennykerr** [comment]
Unfortunately, the latest release also contains invalid metadata (#30).

**Comment by kennykerr** [comment]
I'm preparing an update here: https://github.com/microsoft/windows-rs/pull/2446

It's blocked on a new issue: https://github.com/microsoft/wdkmetadata/issues/31

But depending on what you're actually looking for it may be worth testing this branch so long. Any feedback would be appreciated as I don't have much WDK experience myself.
